DEVELOPERS.md

```
@@ -1,184 +1,186 @@
+Example:
 
 ```bash
-python3 -m pytest tests/ -v --cov --cov-report=term-missing
+pylint src/security/promote_alerts.py
+```
+
+### Expected Output
+
+This is an example of the expected console output after running the tool:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)